presentation/common/src/presentation-common/PresentationRpcInterface.ts

```diff
@@ -218,7 +218,7 @@ export class PresentationRpcInterface extends RpcInterface {
 
   public async getSelectionScopes(_token: IModelRpcProps, _options: SelectionScopeRpcRequestOptions): PresentationRpcResponse<SelectionScope[]> { return this.forward(arguments); }
   // TODO: need to enforce paging on this
-  public async computeSelection(_token: IModelRpcProps, _options: SelectionScopeRpcRequestOptions, _ids: Id64String[], _scopeId: string): PresentationRpcResponse<KeySetJSON> { return this.forward(arguments); }
+  public async computeSelection(_token: IModelRpcProps, _options: SelectionScopeRpcRequestOptions, _ids: Id64String[], _scopeId: string, _scopeParams?: any): PresentationRpcResponse<KeySetJSON> { return this.forward(arguments); }
```

Comment:
Probably unintended: `_scopeParams` is of type `any` here, and `SelectionScopeParams` in RPC implementation.

Reply:
Correct. Couldn't use `@alpha` type in `@public` function, so had to create different overloads. Used the opportunity and made the new overload take a single object argument rather than multiple individual arguments.